src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -34,36 +34,252 @@ import kubernetesCke from 'images/kubernetes_logo_cke.png'
+      },
+      "container_hash": "fryckbos/test@sha256:5b098eaf3c7d5b3585eb10cebee63665b6208bea31ef31a3f0856c5ffdda644b",
+      "container_image": "fryckbos/test:latest",
+      "container_name": "newrelic-logging",
+      "docker_id": "134e1daf63761baa15e035b08b7aea04518a0f0e50af4215131a50c6a379a072",
+      "host": "ip-192-168-17-123.ec2.internal",
+      "labels": {
+        "app": "newrelic-logging",
+        "app.kubernetes.io/name": "newrelic-logging",
+        "controller-revision-hash": "84db95db86",
+        "pod-template-generation": "1",
+        "release": "nri-bundle"
+      },
+      "namespace_name": "nrlogs",
+      "pod_id": "54556e3e-719c-46b5-af69-020b75d69bf1",
+      "pod_name": "nri-bundle-newrelic-logging-jxnbj"
+    },
+    "message": "[2021/09/14 12:30:49] [ info] [engine] started (pid=1)\n",
+    "plugin": {
+      "source": "kubernetes",
+      "type": "fluent-bit",
+      "version": "1.8.1"
+    },
+    "stream": "stderr",
+    "time": "2021-09-14T12:30:49.138824971Z",
+    "timestamp": 1631622649138
+  }
+]
+```
+
+**Minimal Enrichment Log Data** 
+
+```
+[
+  {
+    "cluster_name": "api-test",
+    "container_name": "newrelic-logging",
+    "namespace_name": "nrlogs",
+    "pod_name": "nri-bundle-newrelic-logging-jxnbj",
+    "message": "[2021/09/14 12:30:49] [ info] [engine] started (pid=1)\n",
+    "stream": "stderr",
+    "timestamp": 1631622649138
+  }
+]
+```
+
+
+##### Enable service-level insights, full-body requests, and application profiles through Pixie [#enable-pixie]
+[Pixie](https://docs.px.dev/about-pixie/what-is-pixie/) is an open source observability tool for Kubernetes applications that uses eBPF to automatically collect telemetry data. Users who do not have Pixie installed on their cluster but would like to leverage Pixie's powerful telemetry data collection and visualization on the [New Relic platform](https://docs.newrelic.com/docs/kubernetes-pixie/auto-telemetry-pixie/get-started-auto-telemetry-pixie/), can check `Enable service-level insights, full-body requests, and application profiles through Pixie` in the Kubernetes Integration installation flow for a quick and painless installation. 
+
+###### Community Cloud hosted Pixie is already running on this cluster [#community-cloud-pixie] 
+
+[Pixie can be hosted in various ways](https://docs.px.dev/installing-pixie/install-guides/#title). New Relic provides a different level of integration support for each Pixie hosting option. 
+
+
+**Community Cloud Pixie**
+
+Users who are already leveraging Pixie's Community Cloud can provide an API key to connect Pixie to New Relic. This approach will embed Pixie's live UI into their New Relic account for easy access (via Pixie's Live Debugging tool), as well as write Pixie data into New Relic through the New Relic OpenTelemetry endpoint. 
+
+<img
+  title="service graph in live debugger"
+  alt="service-graph"
+  src={pixieLiveDebugging}
+/>
+
+**Self-Hosted Pixie**
```

Comment:
why do we use these bold formats instead of H2s or H3s or whatever?

Reply:
@zuluecho9 In my local instance, I had moved these items into collapsers. Now that I've pushed this commit up, check the latest.